app.py

```diff
@@ -42,6 +42,7 @@ def index():
 
 if __name__ == "__main__":
     app.run(
-        # host=os.getenv("IP", "0.0.0.0"), port=int(os.getenv("PORT", 8080)),
```

Comment:
Good call we can manually comment this out locally to be able to run the app

Reply:
We'll still should be able to run the app locally without commenting it out